kits/repo-interview-prep/apps/.env.example

```
@@ -0,0 +1,8 @@
+# Environment variables for Repo Interview Prep
+# Copy this file to .env.local and fill in your values from Lamatic Studio → Settings → API Docs
+
+LAMATIC_PROJECT_ENDPOINT=https://your-project-endpoint.lamatic.workers.dev
+LAMATIC_FLOW_ID=your-flow-id-uuid
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_FLOW_ID key should go before the LAMATIC_PROJECT_ENDPOINT key

(UnorderedKey)